src/main/java/com/desiato/puresynth/services/LocalFileStorageService.java

```diff
@@ -0,0 +1,27 @@
+package com.desiato.puresynth.services;
+
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+@Service
+public class LocalFileStorageService implements FileStorageService {
+
+    private static final String BASE_PATH = "audio_files/";
+
+    @Override
+    public String saveFile(byte[] data, String fileName) throws IOException {
```

Comment:
I think it would be better to handle `IOException` here. What could the user do with e.g. `Disk space exceeded` exception or errors like that?